container_files/etc/httpd/conf.d/virt.conf

```diff
@@ -0,0 +1,41 @@
+ServerName dev-mydata
```

Comment:
NOTE:
There are a number of lines that we would want to change for cassie. Particularly the rewrite rules for information to go to the service layer.